prow/config.yaml

```diff
@@ -2454,6 +2454,51 @@ presubmits:
         hostPath:
           path: /mnt/disks/ssd0/docker-graph
 
+  - name: pull-kubernetes-local-e2e
```

Comment:
this is lacking the `skip_report: false` entry that "pull-security-kubernetes-local-e2e" has

I'm not sure which is right, AIUI reporting means it's on you to retrigger until green or /skip if tide ends up watching a PR that has this test trigger

Reply:
added `skip_report: false`

Reply:
`skip_report: ` is up to you as long as things aren't `always_run: true`